.github/workflows/compile-check.yml

```diff
@@ -13,17 +13,18 @@ jobs:
           path: "mods"
       
       - name: Compile Scripts
-        uses: ASpoonPlaysGames/squirrel-re-compiler@v2
+        uses: ASpoonPlaysGames/squirrel-re-compiler@v3
```

Comment:
Uh, where do the changes to CI come from? ^^"

Reply:
no idea lmao

Reply:
Merged another PR into main and merged main into this one and so now it seems to be fine again ^^